src/components/page/ProfilePage.tsx

```diff
@@ -1,10 +1,32 @@
 import { MobileLayout } from '@/components/layouts/MobileLayout';
 import { HeaderMenuCircleButton } from '@/components/parts/Header';
+import { UserProfile } from '@/features/profile/components';
+import { getCurrentUserId } from '@/lib/api/auth/server';
+import { getUserById } from '@/lib/api/user/server';
+
+export const ProfilePage = async () => {
+  /**
+   * Get the user's name.
```

Comment:
Please make it a bit more descriptive.

>  Get the user's name.

This is obvious from the function name.

Here are examples of better versions: 
- Get the user ID based on the auth token of the user currently logged in.
- Get the user ID of the logged-in user based on the auth token.